marketing/templates/marketing/conversion.html

```diff
@@ -20,7 +20,7 @@ <h1 class="page-title__text">Conversion data</h1>
       {% csrf_token %}
       {% if form.non_field_errors %}
       <div class="page-content__inner__wrapper">
-        <ul class="djng-form-errors">
+        <ul class="login-form-errors">
           {% for error in form.non_field_errors %}
           <li class="invalid">{{ error|last }}</li>
```

Comment:
I'm testing this page as I don't think the `last` filter is needed here. (not related to changes in this PR)